dist/index.d.ts

```diff
@@ -11,19 +11,15 @@ interface HTMLCanvasElement {
     contextId:
       | "webgpu"
       | "gpupresent"
```

Comment:
Not a problem with your patch, but "gpupresent" has been deprecated for a while and I believe doesn't even work in Chrome anymore. Could we remove it here and for `OffscreenCanvas` while we're updating this?

Reply:
I openned an issue https://github.com/gpuweb/types/issues/93. We can remove them together with implementation for `OffscreenCanvas` (IIUC)
I will land this as is. Thanks for the review!